packages/blocks/src/api/factory.js

```diff
@@ -117,26 +117,41 @@ const isPossibleTransformForSource = ( transform, direction, blocks ) => {
 		return false;
```

Comment:
I would expect there should be some changes to `api/test/factory.js` for as much code is changing here?

Reply:
I agree. I've been so distracted by e2e tests! 

I've added unit tests for the `from` scenario for wildcard transforms.

I'm struggling with the idea of a `to` transform that accepts wildcards. I suppose there _could_ be a block that can change into any other block?

Reply:
> ideally test cases for the newly introduced functions

Looking at these now.